STYLE.md

```diff
@@ -17,6 +17,7 @@
   includes both const and non-const references.
 * Function names using camel case starting with a lower case letter (e.g., "doFoo()").
 * Struct/Class member variables have a '\_' postfix (e.g., "int foo\_;").
+* Enum values using all CAPS (e.g., "EXTERNAL_ONLY").
```

Comment:
It actually looks like #989 was referring to static const values. Should I hard code PascalCase in the style guide?